### PR TITLE
SONAR-9965 support large rule descriptions

### PR DESCRIPTION
**Pull request by bartfastiel**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NewIndex.java#L361](https://github.com/SonarSource/sonarqube/blob/7a91c46d2c7a03512e787923e8172f63def6f3bb/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L361): Define a constant instead of duplicating this literal "norms" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [NewIndex.java#L362](https://github.com/SonarSource/sonarqube/blob/7a91c46d2c7a03512e787923e8172f63def6f3bb/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L362): Define a constant instead of duplicating this literal "store" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NewIndex.java#L361](https://github.com/SonarSource/sonarqube/blob/71f046af1accafd4d2c2d28c206f06bb342b3d47/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L361): Define a constant instead of duplicating this literal "norms" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [NewIndex.java#L362](https://github.com/SonarSource/sonarqube/blob/71f046af1accafd4d2c2d28c206f06bb342b3d47/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L362): Define a constant instead of duplicating this literal "store" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NewIndex.java#L361](https://github.com/SonarSource/sonarqube/blob/a23576e566a058dce3ee60c93e4aff4e5018dff5/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L361): Define a constant instead of duplicating this literal "norms" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [NewIndex.java#L362](https://github.com/SonarSource/sonarqube/blob/a23576e566a058dce3ee60c93e4aff4e5018dff5/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L362): Define a constant instead of duplicating this literal "store" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: server/sonar-server/src/test/java/org/sonar/server/test/index/TestIndexerTest.java [hunk 1]
```diff
@@ -108,6 +110,32 @@ public void delete_file_by_uuid() throws Exception {
     assertThat(document.get(FIELD_FILE_UUID)).isEqualTo("F2");
   }
 
+  @Test
+  public void long_message() throws Exception {
```
Comment (simonbrandhof) [inline server/sonar-server/src/test/java/org/sonar/server/test/index/TestIndexerTest.java]: could be renamed a bit, for example `long_message_can_be_indexed`.

File: server/sonar-server/src/test/java/org/sonar/server/test/index/TestIndexerTest.java [hunk 2]
```diff
@@ -108,6 +110,32 @@ public void delete_file_by_uuid() throws Exception {
...
+
+    List<SearchHit> hits = getDocuments();
+    Map<String, Object> document = hits.get(0).getSource();
+    assertThat(hits).hasSize(1);
+    assertThat(document.get(FIELD_MESSAGE).toString()).hasSize(50000);
+    assertThat(document.get(FIELD_FILE_UUID)).isEqualTo("F1");
+  }
+
+  @Test
+  public void long_stacktrace() throws Exception {
```
Comment (simonbrandhof) [inline server/sonar-server/src/test/java/org/sonar/server/test/index/TestIndexerTest.java]: same feedback on method name

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NewIndex.java#L361](https://github.com/SonarSource/sonarqube/blob/c6071217db1467bc848e9ad4c19270cbde3f0815/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L361): Define a constant instead of duplicating this literal "norms" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [NewIndex.java#L362](https://github.com/SonarSource/sonarqube/blob/c6071217db1467bc848e9ad4c19270cbde3f0815/server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java#L362): Define a constant instead of duplicating this literal "store" 3 times. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**